menas/ui/generic/formatters.js

```diff
@@ -130,4 +130,11 @@ var Formatters = new function() {
     }
   };
 
+  this.italicMissingProp = function (sStatus) {
+    if (sStatus === 'Missing') {
```

Comment:
This is actually marks the value `Missing` as _Missing_. So it's broken actually,